src/scripts/services/password_manager.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param length a number involving the length of the password
+ * @returns a string of set length containing random charachters
+ */
+export const generatePassword = (length: number): string => {
```

Comment:
This means that if I ask for a password of length 0 (or -10), I get a password with length `1`? Better to have the terminal case be `0` and return an `""` in that case?

Reply:
A separate third-party password generator was implemented